XcodeServerSDK/Server Entities/Issue.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Issue.swift
+//  XcodeServerSDK
+//
+//  Created by Mateusz Zając on 04.08.2015.
+//  Copyright © 2015 Honza Dvorsky. All rights reserved.
+//
+
+import Foundation
+
+public class Issue: XcodeServerEntity {
+    
+    public enum Type: String {
+        case BuildServiceError = "buildServiceError"
+        // Looks like build service warnings are treated as warnings in case
+        // of type. In case of Issue Type they're Build Service Error
+        // case BuildServiceWarining = "warning"
+        case TriggerError = "triggerError"
+        case Error = "error"
+        case Warning = "warning"
+        case TestFailure = "testFailure"
+        case AnalyzerWarning = "analyzerWarning"
```

Comment:
Hmm but Xcode shows `build service warning` and `warning` as two separate issues, doesn't it? And it uses the same API, so they have a way to tell them apart. Remind me again what the two look like (feel free to paste a sample of each). There must be a way to tell them apart.